tensorflow_probability/python/mcmc/replica_exchange_mc_test.py

```diff
@@ -804,8 +825,8 @@ def target_log_prob(x, y):
       # z = matmul(inv(chol(true_cov)), [x, y] - true_mean)
       xy = tf.stack([x, y], axis=-1) - true_mean
       z = linop.solvevec(xy)
-      return -0.5 * tf.reduce_sum(z**2., axis=-1)
 
+      return -0.5 * tf.reduce_sum(z**2., axis=-1)
```

Comment:
Typo here?  There should be a blank between the end of the previous function and the start of the next one, anyway.